manager/state/doc.go

```diff
@@ -20,8 +20,8 @@
 //		return fmt.Errorf("transaction failed: %v", err)
 //	}
 //
-// WatchableStore is a version of Store that exposes watch functionality.
-// These expose a publish/subscribe queue where code can subscribe to
+// MemoryStore is a version of Store that exposes watch functionality.
```

Comment:
The `Store` interface no longer exists, so it shouldn't be referenced here.

It is also referenced above.

Could you please move this file to `manager/state/store` and change the references to `Store` to `MemoryStore`?

Note that the `state.Tx` in the example is out of date and should be `store.Tx`.